ignite/handlers/param_scheduler.py

```diff
@@ -226,6 +227,256 @@ def plot_values(cls, num_events: int, **scheduler_kwargs: Mapping) -> Any:
         return ax
 
 
+class StateParameterScheduler(ParamScheduler):
+    """An abstract class for updating a state parameter during
+    training not belonging to an Optimizer parameter group.
+
+    Args:
+        parameter_setter: function that sets the required parameter value.
+        param_name: name of parameter to update.
+        save_history: whether to log the parameter values to
+            `engine.state.param_history`, (default=False).
+
+    .. versionadded:: 0.6.0
+    """
+
+    def __init__(
+        self, parameter_setter: Callable, param_name: str, save_history: bool = False,
+    ):
+        self.param_name = param_name
+        self.event_index = 0
+        self._save_history = save_history
+        self.parameter_setter = parameter_setter
+        self._state_attrs = ["event_index", "param_name", "save_history", "parameter_setter"]
+
+    def __call__(self, engine: Optional[Engine], name: Optional[str] = None) -> None:
```

Comment:
`name` is unused here ?

Reply:
It seems to be mainly used by `ParamGroupScheduler`, and maybe not needed for the state parameters. Do you think we should stick to `ParamScheduler` API ?

Reply:
Ok, let's keep it BC compatible, sorry for the noise.